src/ctia/stores/es/init.clj

```diff
@@ -82,25 +84,42 @@
       (log/error "cannot update mapping. You probably tried to update the mapping of an existing field. It's only possible to add new field to existing mappings. If you need to modify the type of a field in an existing index, you must perform a migration" (ex-data e))
       (system-exit-error))))
 
+(defn get-existing-indices
+  [conn index]
+  ;; retrieve existing indices using wildcard to identify ambiguous index names
+  (let [existing (-> (es-index/get conn (str index "*"))
+                     keys
+                     set)
+        index-pattern (re-pattern (str index "(-\\d{4}.\\d{2}.\\d{2}.*)?"))
+        matching (filter #(re-matches index-pattern (name %))
+                         existing)
+        ambiguous (difference existing (set matching))]
+    (if (seq ambiguous)
+      (do (log/warn (format "ambiguous index names, %s and %s."
```

Comment:
* `%s and %s` -> `index: %s,  ambiguous: %s`
* Always pass `pr-str` before serializing items to be logged/println-ed
  * else, e.g. `(format "%s" nil)` returns `"null"` instead of `"nil"`